src/otx/core/model/base.py

```diff
@@ -410,8 +410,7 @@ def load_state_dict_incrementally(self, ckpt: dict[str, Any], *args, **kwargs) -
             msg = "Checkpoint should have `label_info`."
             raise ValueError(msg, ckpt_label_info)
 
-        if isinstance(ckpt_label_info, dict):
-            ckpt_label_info = LabelInfo(**ckpt_label_info)
```

Comment:
Could you add a snapshot loading test for sseg as well?

Reply:
https://github.com/open-edge-platform/training_extensions/pull/4443/commits/55041e903e46c12c3bd772ffa3ffa369cb791832